QGIS_SERVER_TRUST_LAYER_METADATA setting environnement (#6014)
qgis/QGIS-Documentation: docs/server_manual/config.rst

In this case, "bad" layers will just be ignored, and the project will be considered
valid and available.

QGIS_SERVER_TRUST_LAYER_METADATA
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Trust layer metadata is an option at the project level to improve project read time by using vector layer's extent defined in the project and disabling the check primary key unicity for Postgres/Postgis layers.

Reviewer: ```suggestion
This is an option at the project level to improve project read time by using the vector
layer extents defined in the project metadata and disabling the check for
PostgreSQL/PostGIS layer primary key uniqueness.
```

In this case, "bad" layers will just be ignored, and the project will be considered
valid and available.

QGIS_SERVER_TRUST_LAYER_METADATA
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

This is an option at the project level to improve project read time by using the vector
layer extents defined in the project metadata and disabling the check for
PostgreSQL/PostGIS layer primary key uniqueness.

> The trust layer metadata can be forced by setting this variable to ``1`` or ``true``.
> In this case, the vector layer's extent will be the one defined in the project and Postgres/Postgis layers' primary key defined in the datasource are considered as unique without check.

Reviewer: ```suggestion
Trusting layer metadata can be forced by setting this variable to ``1`` or ``true``.
The vector layer's extent will then be the one defined in the project, and the
PostgreSQL/PostGIS layer's primary key defined in the data source is
considered as unique without a check.
```
